### PR TITLE
FWI-940 - OPA should not panic on missing kind manifest files

### PR DESCRIPTION
**Pull request by makoscafee**
This PR fixes #

## Checklist
* [ ] I have signed the CLA
* [ ] I have updated/added any relevant documentation

### What alternative solution should we consider, if any?

[Internal Ticket FWI-940](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-940)

### CODE REVIEW

**Review by ivanfetch-fw — APPROVED** [review]
@makoscafee How does CI behave when continuing to process a manifest which is missing a Kind?

File: plugins/ci/pkg/ci/opa.go [hunk 1]
```diff
@@ -56,14 +56,20 @@ func (ci CIScan) ProcessOPA(ctx context.Context) (models.ReportInfo, error) {
...
+					for _, node := range nodes {
+						nodeMap := node.(map[string]interface{})
+						files = append(files, nodeMap)
+					}
+				} else {
+					files = append(files, yamlNode)
 				}
 			} else {
+				logrus.Warn("Manifest is missing field Kind")
 				files = append(files, yamlNode)
```
Comment (rbren) [inline plugins/ci/pkg/ci/opa.go:73]: If Kind is missing, we should skip this file.

Comment (makoscafee) [reply plugins/ci/pkg/ci/opa.go]: aah gotcha!

**Comment by makoscafee** [comment]
> @makoscafee How does CI behave when continuing to process a manifest which is missing a Kind?

I think the idea is to ignore them, and not process them at all.

**Pull request merged**